portfolio/src/main/webapp/script.js

```diff
@@ -21,6 +21,11 @@ function addRandomFunFact() {
     'I also have a hydroponic garden.', 
     'My favorite cereal is Cheerios.', 
     'When I was in third grade I refused to wear anything other than turtle-neck shirts.',
+    'I\'ve picked up the hobby of making my own hot sauces.',
+    'I have an extensive collection of cheap pens.',
```

Comment:
What's the best cheap pen? I've always had a fondness for the Pilot G2

Reply:
The G2 is great! The 0.38mm version is my favorite cheap, widely available pen. The Zebra Sarasa is another really good choice, but it's less common to find in stores.